packages/contracts/test/OwnershipTest.js

```diff
@@ -79,7 +73,7 @@ contract('All Liquity functions with onlyOwner modifier', async accounts => {
 
   describe('TroveManager', async accounts => {
     it("setAddresses(): reverts when called by non-owner, with wrong addresses, or twice", async () => {
-      await testSetAddresses(troveManager, 11)
+      await testSetAddresses(troveManager, 10)
```

Comment:
what does this (and the 7=>6 below) change indicate?

Reply:
its caused by _lqtyTokenAddress being removed from the setAddresses call